### PR TITLE
Set ansible_python_interpreter

### PR DESCRIPTION
**Pull request by williambelle**
As per [this § of the shiny new suitcase documentation](https://github.com/epfl-si/ansible.suitcase/blob/master/COOKBOOK.md#using-the-suitcases-python-for-your-local-tasks)

### CODE REVIEW

**Comment by williambelle** [comment]
See https://github.com/epfl-si/ansible.suitcase/blob/master/COOKBOOK.md#using-the-suitcases-python-for-your-local-tasks

File: ansible/elasticsible [hunk 1]
```diff
@@ -61,6 +61,7 @@ mode=ansible-playbook
 inventory_mode="test"
 
 declare -a ansible_args
+ansible_args=(-e "elasticsible_dir=$PWD/ansible-tmp")
```
Comment (domq) [inline ansible/elasticsible:64]: `elasticsible_dir` → `elasticsible_suitcase_dir`?

File: ansible/host-prod [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [wwp-group]
-wwp      ansible_connection=local   openshift_namespace=wwp
\ No newline at end of file
+wwp      ansible_connection=local  ansible_python_interpreter={{elasticsible_dir}}/bin/python3  openshift_namespace=wwp
```
Comment (domq) [inline ansible/host-prod:2]: Same

File: ansible/host-test [hunk 3]
```diff
@@ -1,2 +1,2 @@
 [wwp-test-group]
-wwp-test      ansible_connection=local   openshift_namespace=wwp-test
\ No newline at end of file
+wwp-test      ansible_connection=local  ansible_python_interpreter={{elasticsible_dir}}/bin/python3  openshift_namespace=wwp-test
```
Comment (domq) [inline ansible/host-test:2]: Same

**Review by domq — CHANGES_REQUESTED** [review]
Sorry to be that guy, but...

**Review by domq — APPROVED** [review]

**Pull request merged**